packages/runner/webpack.config.ts

```diff
@@ -85,8 +85,7 @@ mainConfig.resolve = {
 
 // @ts-ignore
 const crossOriginConfig: webpack.Configuration = {
-  mode: 'production',
```

Comment:
Was this intentional?

Reply:
I think we still want to keep the simple config if I remember correctly

Reply:
Yes, as discussed in standup today, we want to use the same config as `cypress_runner` which uses `commonConfig` to ensure we are getting sourcemaps for development.